openlibrary/templates/site/head.html

```diff
@@ -54,4 +54,8 @@
     $if query_param("_firebug") == "true":
         <script type='text/javascript' src='http://getfirebug.com/releases/lite/1.2/firebug-lite-compressed.js'></script>
 
+    $if '/account/create' in request.canonical_url:
```

Comment:
Is this code already being included elsewhere to power v1 recaptcha?

Reply:
Which all pages is recaptcha v1 being used?